### DISCUSSION TITLE
quick/nas

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#570] [Announcements]
# quick/nas

系统兼容性 群晖、威廉通等...; 准备工作 1. 安装Docker An image 2. 开启SSH An image 自动安装 填写正确的NAS信息，点击安装星空即可 NAS自动安装程序 - 点我下载【下载后解压运行】 (https://skynet-beijing.oss-cn-beijing.aliyuncs.com/stars/releas...

https://ip4.ink/quick/nas.html

**Comment by 912988434** [comment]
威廉通=威联通,错别字

**Reply by ld-2022** [reply]
感谢反馈，马上修改

**Reply by 912988434** [reply]
建议版本更新记录里加上每次发布的时间，另外自定义节点取消了么，看文档中没了介绍

**Reply by ld-2022** [reply]
好的，这个发版时间我下次加上，自定义节点没有取消，文档刚迁移到这里还没来得及更新上

**Reply by 912988434** [reply]
很强，我看有 qq群，微信群有么

**Reply by ld-2022** [reply]
有的，你加我微信，我把你拉进去：zxw18511511011

**Reply by Mister-Hope** [reply]
# See readme, you are not allowed to use my repo in production. Use your own repo discussion, otherwise we will block you.
